<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>hetio &middot; Disease-Associated Genes</title>
  <meta name="description" content="Prioritizing Disease-Associated Genes with Heterogeneous Network Edge Prediction.">
  <link href="../favicon.ico" rel="icon" type="image/x-icon" />

  <!-- Bootstrap core CSS -->
  <link href="css/bootstrap.min.css" rel="stylesheet">
  <!-- Custom styles for this template -->
  <link href="css/bootstrap-theme.css" rel="stylesheet">

  <style type="text/css" title="currentStyle">
    .glyphicon-rat {
      background-image: url("./icons/rat-14x14.png");
      background-position: center center;
      height: 14px;
      width: 14px;
    }
    .glyphicon-bug {
      background-image: url("./icons/bug-14x14.png");
      background-position: center center;
      height: 14px;
      width: 14px;
    }
  </style>

</head>

<body>

  <!-- Main jumbotron for a primary marketing message or call to action -->
  <div class="jumbotron">
    <div class="container">
      <h1>Prioritizing Disease-Associated Genes
        <small>with Heterogeneous Network Edge Prediction.</small>
      </h1>
    </div>
  </div>


  <div class="container">
    <div class="row">
      <!-- HNEP -->
      <div class="col-md-12">
        <h2>About</h2>
          <p>The first decade of Genome Wide Association Studies (GWAS) has uncovered a wealth of disease-associated variants. Two important derivations will be the translation of this information into a multiscale understanding of pathogenic variants, and leveraging existing data to increase the power of existing and future studies through prioritization. We explore edge prediction on heterogeneous networks&#8212;graphs with multiple node and edge types&#8212;for accomplishing both tasks.</p>
          <p>First we constructed a network with 18 node types&#8212;genes, diseases, tissues, pathophysiologies, and 14 MSigDB (molecular signatures database) collections&#8212;and 19 edge types from high-throughput publicly-available resources. From this network composed of 40,343 nodes and 1,608,168 edges, we extracted features that describe the topology between specific genes and diseases. Next, we trained a model from GWAS associations and predicted the probability of association between each protein-coding gene and each of 29 well-studied complex diseases. The model, which achieved 132-fold enrichment in precision at 10% recall, outperformed any individual domain, highlighting the benefit of integrative approaches. We identified pleiotropy, transcriptional signatures of perturbations, pathways, and protein interactions as fundamental mechanisms explaining pathogenesis.</p>
          <p>
            <a class="btn btn-default" title="Himmelstein &amp; Baranzini (2015) PLOS Computational Biology" href="https://doi.org/10.1371/journal.pcbi.1004259" role="button">Publication »</a>
          </p>
      </div>
    </div>

    <div class="row">
      <!-- Disease-Associated Genes -->
      <div class="col-md-12">
        <h2>Browse</h2>
        <p>View the results of our analysis. Browse predictions by gene or disease and learn more about the network-based features. Results include context-specific summaries and performance assessments. Applications of interest include:
<ul>
  <li>Using the predications as prior probabilities of association to increase the power of GWAS analysis.</li>
  <li>Determining candidate causal genes within genomic regions identified by GWAS.</li>
  <li>Identifying genes of biological interest for a specific disease.</li>
  <li>Identifying diseases of interest for a specific gene.</li>
  <li>Exploring a high-confidence, gene-centric translation of the GWAS Catalog.</li>
  <li>Comparing the informativeness of various information domains for identifying pathogenic variants.</li>
  <li>Viewing the contibution of each network-based feature composing an association prediction.</li>
</ul>

</p>
        <p>
          <a class="btn btn-default" href="./browse" role="button">View details »</a>
        </p>
      </div>
    </div>

    <div class="row">
      <div class="col-md-6">
        <h2>Downloads</h2>
        <p>Download an extensive collection of files related to the project. Available downloads include our predictions, positive set of disease-associated genes, vector graphics, serialized networks, and processed data sources.</p>
        <p>
          <a class="btn btn-default" href="./downloads" role="button">View details »</a>
        </p>
      </div>
      <div class="col-md-6">
        <h2>Media</h2>
        <p>Browse the collection of media relating to the project. Check back here for citation information.</p>
        <p>
          <a class="btn btn-default" href="../media" role="button">View details »</a>
        </p>
      </div>

    </div>
  </div>


  <script type="text/javascript" charset="utf-8" src="js/jquery-2.1.0.min.js"></script>
  <script src="js/bootstrap.min.js"></script>

  <!-- Piwik -->
  <script type="text/javascript">
    var _paq = _paq || [];
    _paq.push(['trackPageView']);
    _paq.push(['enableLinkTracking']);
    (function() {
      var u="//piwik.dhimmel.com/";
      _paq.push(['setTrackerUrl', u+'piwik.php']);
      _paq.push(['setSiteId', 3]);
      var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
      g.type='text/javascript'; g.async=true; g.defer=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
    })();
  </script>
  <noscript><p><img src="//piwik.dhimmel.com/piwik.php?idsite=3" style="border:0;" alt="" /></p></noscript>
  <!-- End Piwik Code -->

</body>

</html>
